spec/knockout-fast-foreach-spec.js

```diff
@@ -24,6 +24,7 @@ function setupSynchronousFrameAnimation () {
 }
 
 describe("applying bindings", function () {
+  FastForEach.DEBUG = true;
```

Comment:
This should be set to `true` only in those cases where it's relevant i.e. we should be testing non-debug in the ordinary course.  Not that it's a big deal here, but it's usually good design.